src/applications/mhv-secure-messaging/components/ThreadList/ThreadsList.jsx

```diff
@@ -18,8 +24,14 @@ const ThreadsList = props => {
     threadsPerPage,
   } = props;
 
+  const { INBOX } = DefaultFolders;
+
   const location = useLocation();
 
+  const removeLandingPageFF = useSelector(
+    state => state.featureToggles.mhv_secure_messaging_remove_landing_page,
```

Comment:
I believe we are encouraged to use `featureFlagNames.json` here to look up the flag from state